projects/plugins/crm/modules/portal/templates/invoices.php

```diff
@@ -41,17 +38,17 @@
 }
 </style>
 <div class="alignwide zbs-site-main zbs-portal-grid">
-    <nav class="zbs-portal-nav">
-		<?php $portal->render->portal_nav($invoice_endpoint); ?>
-    </nav>
-    <div class='zbs-portal-content zbs-portal-invoices-list'>
-        <h2><?php esc_html_e('Invoices','zero-bs-crm'); ?></h2>
-        <div class='zbs-entry-content zbs-responsive-table' style="position:relative;">
-		    <?php 
-				$invoices_endpoint = new Automattic\JetpackCRM\Invoices_Endpoint( $portal );
-				$invoices_endpoint->list_invoices_html_output();
-			 ?>
-        </div>
-    </div>
-    <div class="zbs-portal-grid-footer"><?php $portal->render->portal_footer(); ?></div>
+	<nav class="zbs-portal-nav">
+	<?php $portal->render->portal_nav( $invoice_endpoint ); ?>
+	</nav>
+	<div class='zbs-portal-content zbs-portal-invoices-list'>
```

Comment:
Is the idea to be consistent and use double quotes for class names? If so, this particular line is different to the others.

```suggestion
	<div class="zbs-portal-content zbs-portal-invoices-list">
```

Reply:
Indeed, double quotes for classes is secondary but preferred, so thanks for the catch! I've applied all three suggestions.